Engine/Makefile-defs.linux

```diff
@@ -1,8 +1,13 @@
+FT_CFLAGS = $(shell pkg-config --cflags freetype2)
```

Comment:
Probably a good idea to use `:=` because they'll only be expanded once.

Reply:
oh right. oversight on my side